### PR TITLE
Julio review of module 3

### PR DESCRIPTION
**Pull request by juliofdiaz**
@natacha-couto I have reviewed module 3. I am on the fence in terms of adding the pipeline graph showing the assembly step. Could you please add Nabil as a contributor and assign this to him?

### CODE REVIEW

**Comment by juliofdiaz** [comment]
I've added Nabil, once he accepts. I'll comment him in for his support

File: module3/module3.qmd [hunk 1]
```diff
@@ -16,15 +16,17 @@ title: "SESSION 3 - Introduction to Trimming, Assembly and QC"
 
 *Note: This exercise is a work of fiction. Any resemblance to a real life situation is purely coincidental.*
 
-In December 2020, three cases of primary bloodstream infections sustained by ertapenem-resistant *Klebsiella pneumoniae* were observed over a two-month period in the neonatal intensive care unit (NICU) at the University College Hospital in Ibadan, Oyo State. The NICU consists of a 15-bed tertiary level ward. Faecal screening of all admitted neonates was performed. Ten additional samples from the environment were collected. A plan of the NICU and sample collection can be seen below.
+In December 2020, three cases of primary bloodstream infections sustained by ertapenem-resistant *Klebsiella pneumoniae* were observed over a two-month period in the neonatal intensive care unit (NICU) at the University College Hospital in Ibadan, Oyo State. This NICU consists of a 15-bed tertiary level ward. Since the detectionf of these infections, faecal screening of all admitted neonates was performed. Additionally, ten samples from the environment were collected. The epidemiological and demographic data is summarised in the file [epi_data.csv](data/epi_data.csv).
```
Comment (natacha-couto) [inline module3/module3.qmd]: change two-month period to two-week period. detection has an extra "f" at the end that needs to be removed.

File: module3/module3.qmd [hunk 2]
```diff
@@ -16,15 +16,17 @@ title: "SESSION 3 - Introduction to Trimming, Assembly and QC"
...
 
-<img src="img/unit.png" width="70%" height="70%"/>
+<img src="img/unit.png" width="70%" height="70%"/><br/>
+Plan of the NICU. Locations of sample collection is indicated with light blue circles.
+<br/><br/>
 
-The reference lab has sent you the culture results in an Excel file called ([`lab_results.xlsx`](data/lab_results.xlsx)). Faecal screening of all admitted neonates, revealed that **five other neonates** were carriers of *K. pneumoniae*. The lab reported that **2 out of 10** environmental samples were positive for *K. pneumoniae*. **This immediately prompted the closure of 1 room.**
+The reference lab sent you the culture results from the faecal screening and the environmental sampling in an Excel file called ([`lab_results.xlsx`](data/lab_results.xlsx)). Faecal screening of the admitted neonates, revealed that **five other neonates** were carriers of *K. pneumoniae*. Also, the lab reported that **2 out of 10** environmental samples were positive for *K. pneumoniae*. **This immediately prompted the closure of 1 room.**
```
Comment (natacha-couto) [inline module3/module3.qmd]: not only faecal screening and environmental samples, the bloodstream strains as well.

File: module3/module3.qmd [hunk 3]
```diff
@@ -16,15 +16,17 @@ title: "SESSION 3 - Introduction to Trimming, Assembly and QC"
...
 
-One colony from each confirmed case and two from the environment were sent to the GHRU-Ibadan unit for sequencing. The epidemiological and demographic data is summarised in the file [epi_data.csv](data/epi_data.csv).
+One colony from each confirmed case and from each of Ithe two positive environmental samples were sent to the GHRU-Ibadan unit for sequencing.
```
Comment (natacha-couto) [inline module3/module3.qmd]: an extra "I" on "the".

File: module3/module3.qmd [hunk 4]
```diff
@@ -37,14 +39,154 @@ At the end of this session the participants will be able to:
...
+## Data
+
+You will find 11 sets of paired-end Illumina reads in the `/data/`<@Nabil please check the raw data is here> directory. Thus, this directory will contain 22 fastq files (two per genome). You can check the data is actually there:
+
+```bash
+# Check the contents of the /data/ directory
+ls /data/
+```
+
+The /data directory will serve as the input directory for the GHRU assembly pipeline. You can practice fetching the data from a remote location such as Figshare, ENA or SRA folling the instructions in the [fetching data](#fetching-data)  bonus section.
```
Comment (natacha-couto) [inline module3/module3.qmd]: error in "following"

File: module3/module3.qmd [hunk 5]
```diff
@@ -37,14 +39,154 @@ At the end of this session the participants will be able to:
...
+```
+
+You may want to import the quality report (`results/quality_reports/qualifyr_report.tsv`) to your local machine so as to explore whether the assemblies are of high quality or not. You can use the `scp` command to do this:
+
+```bash <@Nabil please provide a proper username:address combination>
+# Copy the quality report to your local machine
+scp user@ghru-assembly:/results/quality_reports/qualifyr_report.tsv /path/to/local/machine
+```
+
+You can use the same commad to copy the assemblies to your local machine (hint: `scp -r` helps you move an entire directory).
```
Comment (natacha-couto) [inline module3/module3.qmd]: it should be command

File: module3/module3.qmd [hunk 6]
```diff
@@ -37,14 +39,154 @@ At the end of this session the participants will be able to:
...
+scp user@ghru-assembly:/results/quality_reports/qualifyr_report.tsv /path/to/local/machine
+```
+
+You can use the same commad to copy the assemblies to your local machine (hint: `scp -r` helps you move an entire directory). 
+
+# Bonus Materials
+
+## Fetching data {#fetching-data}
+
+You can use tools sucha as `wget` or `curl` to fetch data from a remote location such as Figshare, ENA or SRA. For example, you can fetch the data from the following Figshare link:
```
Comment (natacha-couto) [inline module3/module3.qmd]: it should be such not "sucha"

**Review by natacha-couto — CHANGES_REQUESTED** [review]
changes are needed

File: module3/data/~$lab_results.xlsx
Comment (natacha-couto) [inline module3/data/~$lab_results.xlsx:1]: erase this file please

**Review by natacha-couto — CHANGES_REQUESTED** [review]
erase file

**Pull request closed without merging**